chapter_4_practic/task6.cpp

```diff
@@ -0,0 +1,74 @@
+﻿// We said earlier that C++ I / O statements don’t automatically understand the data types of
+// enumerations.Instead, the(>> ) and (<< ) operators think of such variables simply as integers.You can overcome this limitation by using switch statements to translate between
+// the user’s way of expressing an enumerated variableand the actual values of the enumerated variable.For example, imagine an enumerated type with values that indicate an
+// employee type within an organization :
+// enum etype { laborer, secretary, manager, accountant, executive,researcher};
+// Write a program that first allows the user to specify a type by entering its first letter
+// (‘l’, ‘s’, ‘m’, and so on), then stores the type chosen as a value of a variable of type
+// enum etype, and finally displays the complete word for this type.
+// Enter employee type(first letter only)
+// laborer, secretary, manager,
+// accountant, executive, researcher): a
+// Employee type is accountant.
+// You’ll probably need two switch statements : one for inputand one for output.
+
+#include <iostream>
+
+int main()
+{
+	char ch = ' ';
+	enum etype { laborer, secretary, manager, accountant, executive, researcher };
```

Comment:
prefer the usage of 
enum class
https://en.cppreference.com/w/cpp/language/enum

Reply:
fixed in https://github.com/Dmitriy-Romanchuk/master/pull/4/commits/2048c13f717dd24c8d8d80020596b020faed2a90